docs/edit-tool.md

```diff
@@ -154,6 +155,14 @@ In Addtional to the default settings for a tool, you can specify the following o
 
 ![Route Planner Panel](route-planner.png)
 
+## Geomark
+
+The Geomark tool allows creating and loading geomarks. A geomark is an area of interest that is stored in a web service and can be shared by a URL. A geomark can be created in SMK by drawing one or more shapes and saving the drawing as a geomark. A geomark can also be loaded by its URL.
+
+### Enable Link to Create a Geomark from a File
+
+When checked, `Enable Link to Create a Geomark from a File` will add an option to the panel that opens a browser tab to the page of the geomark web service where a file can be uploaded to create a new geomark. The geomark's URL can then be used to load a geomark in SMK. This is an alternative to drawing a shape in SMK and saving it as a geomark.
```

Comment:
Is this still a hyperlink? Or a button?

Reply:
Good catch. It's a button. I'll replace "link" with "option" to reflect the language we use in the Geomark configuration page (edit-tool-details-geomark.html).